nixos/tests/all-tests.nix

```
@@ -411,6 +411,7 @@ in
   domination = handleTest ./domination.nix { };
   dovecot = handleTest ./dovecot.nix { };
   drawterm = discoverTests (import ./drawterm.nix);
+  draupnir = runTest ./matrix/draupnir.nix;
```

```suggestion
  draupnir = runTest ./matrix/draupnir.nix;
  draupnir-with-pantalaimon = runTest {
    imports = [ ./matrix/draupnir.nix };
    _module.args.withPantalaimon = true;
  };
```

That way we can provide a variant for both scenarios.